app/components/Sidebar/components/SidebarLink.tsx

```diff
@@ -194,8 +196,17 @@ const Link = styled(NavLink)<{
   ${(props) =>
     props.$isDraft &&
     css`
-      padding: 4px 14px;
```

Comment:
Now that the update is seamless it made it obvious that drafts are slightly differently sized on desktop